pages/project-version/middleware/index.js

```diff
@@ -175,17 +175,14 @@ const getAllChanges = () => (req, res, next) => {
     getGrantedVersion(req)
   ])
     .then(([previousVersion, grantedVersion]) => {
-      return Promise.all([
-        getChanges(req.version, previousVersion),
-        getChanges(req.version, grantedVersion)
-      ]);
-    })
-    .then(([latest, granted]) => {
-      res.locals.static.changes = {
-        latest,
-        granted
+      return {
+        latest: getChanges(req.version, previousVersion),
+        granted: getChanges(req.version, grantedVersion)
```

Comment:
This bit is just removing `Promise.all` because the function isn't async so it was complicating things for no reason.